src/server/actions/org.actions.ts

```diff
@@ -62,6 +64,55 @@ export const getReviewerPublicKeyAction = userAction(async () => {
 
 export type OrgUserReturn = Awaited<ReturnType<typeof getUsersForOrgAction>>[number]
 
+const updateOrgSettingsSchema = z
+    .object({
+        orgSlug: z.string(),
+    })
+    .merge(orgSchema.pick({ name: true, description: true }))
+
+export const updateOrgSettingsAction = orgAdminAction(async ({ orgSlug, name, description }) => {
+    // Fetch the current organization
+    const currentOrg = await db.selectFrom('org').select('name').where('slug', '=', orgSlug).executeTakeFirst()
+    if (!currentOrg) {
+        throw new ActionFailure({ org: `Organization with slug ${orgSlug} not found.` })
+    }
+    const originalName = currentOrg.name
+
+    // Update the database first, Clerk second
+    await db.updateTable('org').set({ name, description }).where('slug', '=', orgSlug).executeTakeFirstOrThrow()
+
+    try {
+        // Update Clerk
+        const clerk = await clerkClient()
+        await clerk.organizations.updateOrganization(orgSlug, { name })
+    } catch (error) {
+        console.error(`Failed to update organization name in Clerk for ${orgSlug}:`, error)
+        // Revert the database change
+        try {
+            await db
+                .updateTable('org')
+                .set({ name: originalName })
+                .where('slug', '=', orgSlug)
+                .executeTakeFirstOrThrow()
+            console.warn(`Successfully reverted organization name in DB for ${orgSlug} after Clerk update failure.`)
```

Comment:
we should replace console.warn with a new logger that calls sentry.captureMessage as well as logging the message.  Maybe we can extend lib/logger to do that for error and warning?

Reply:
I will replace all console functions here with logger.ts functions. I agree with the sentry captureMessage. But I will leave that for a new PR after this one (because it requires some refactoring of logger.ts)